docs/reference/technology/USB.md

```diff
@@ -2,11 +2,11 @@
 
 ## Introduction
 
-A functional Mbed OS USB device consists of three parts - an implementation of USBPhy, the USBDevice stack and a USB component code:
+A functional Mbed OS USB device consists of three parts: an implementation of USBPhy, the USBDevice stack and a USB component code.
```

Comment:
We like to keep colons before bulleted lists, but I don't like two colons in a row. What do you suggest?

Reply:
I changed this so the double colon was no longer necessary, but there's another one below that I can't do that kind of restructure with.